### ISSUE TITLE
Test example project more thoroughly on Windows (handling of whitespace in file system path)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cstroe/fitnesse-launcher-maven-plugin#11]
```
Currently, if the project is run from somewhere under C:\Documents and 
Settings\username\project,
it will fail

```

Original issue reported on code.google.com by `stewart@javahelp.co.uk` on 12 Apr 2013 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
See docs at 
http://fitnesse-launcher-maven-plugin.googlecode.com/svn/maven/site/fitnesse-lau
ncher-maven-plugin/faq.html#whitespace
```

Original comment by `stewart@javahelp.co.uk` on 9 Jul 2013 at 9:19

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in FitNesse versions 20140623 onwards.
https://github.com/unclebob/fitnesse/issues/278

```

Original comment by `stewart@javahelp.co.uk` on 8 Jul 2014 at 10:51
